test/Suites/Plutus/Model/Script/V1/Test/Counter.hs

```diff
@@ -91,7 +91,7 @@ increment :: PubKeyHash -> Integer -> Run Bool
 increment pkh inc = checkBalance (pkh `owns` mempty <> counterScript `owns` mempty) $ do
   utxos <- utxoAt counterScript
   let [(counterRef, counterOut)] = utxos
-  mDat <- datumAt @CounterDatum counterRef
+  mDat <- datumAt @_ @CounterDatum counterRef
```

Comment:
Use Monad parameter as second argument in this function and everywhere else
to avoid `@_`. As script type changes more often than monad type